app/services/force/flagged_record_set_service.rb

```diff
@@ -67,7 +67,7 @@ def update_flagged_application(data)
       return nil unless data[:Id]
       puts "updating flagged application with: #{data.as_json}"
       # by passing data[:Id] it will know to update that record
-      @client.update('Flagged_Application__c', data)
+      @client.update!('Flagged_Application__c', data)
```

Comment:
this forces it to raise an exception if it fails that has an error message in it instead of just silently returning false